pages/index.js

```diff
@@ -1,7 +1,187 @@
+					<div className='index--slider'>
+						<div className='index--ft__wrapper'>
+							<Triangle></Triangle>
+							<h3 className='index--flavortext'>Digital Dominance</h3>
+						</div>
+						<h2>Resume</h2>
+						<p className='p--resume'>
+							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam
+							eget erat luctus, condimentum lectus eu, egestas tortor.
+						</p>
+					</div>
+					<ul className='index--ul'>
+						<li>01.-----</li>
+						<li>02.---</li>
+						<li>03.---</li>
+					</ul>
+				</div>
+				<div className='index--second'>
+					<div className='index--services'>
+						<Triangle></Triangle>
+						<h3 className='index--services'>Digital Dominance</h3>
+						<h2>Develop Digital Products</h2>
+						<p>
+							Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.
+						</p>
+						<Link href='#'>
+							<a>View All ➡️</a>
+						</Link>
+					</div>
+					<Service
+						imageSrc='https://picsum.photos/200/300'
+						imageAlt='#'
+						text='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.'
+						link='#'
+						title='Web Development'
+					/>
+					<Service
+						imageSrc='https://picsum.photos/200/300'
+						imageAlt='#'
+						text='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.'
+						link='#'
+						title='SEO Optomization'
+					/>
+					<Service
+						imageSrc='https://picsum.photos/200/300'
+						imageAlt='#'
+						text='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+							fringilla facilisis tortor non eleifend. Ut.'
+						link='#'
+						title='Ui/UX Design'
+					/>
+				</div>
+				<div className='index--third'>
+					<Triangle></Triangle>
+					<h3 className='index--portfolio'>Portfolio</h3>
+					<h2>Check Our Latest Cases</h2>
+					<Link href='#'>
+						<a>View All ➡️</a>
+					</Link>
+					<Image
+						src='https://picsum.photos/200/300'
+						width='200'
+						height='300'
+						alt='rando'
+					></Image>
+					<Image
+						src='https://picsum.photos/200/300'
+						width='200'
+						height='300'
+						alt='rando'
+					></Image>
+					<Image
+						src='https://picsum.photos/200/300'
+						width='200'
+						height='300'
+						alt='rando'
+					></Image>
+				</div>
+				<div className='index--fourth'>
+					<h2 className='cases'>CASES</h2>
+					<h3>cases</h3>
+					<h2>Always be in touch</h2>
+					<label htmlFor='email'>email</label>
+					<input type='email' id='email' />
+					<label htmlFor='checkbox'>I accept all Terms and Conditions</label>
+					<input type='checkbox' id='checkbox' />
+				</div>
+				<div className='fifth'>
+					<h1>DIGITAL</h1>
+					<h2>DIGITAL</h2>
+				</div>
+				<div className='index--sixth'>
+					<div className='index-whatwedo'>
+						<Triangle />
+						<h3 className='index--digital__flavortxt'>What We Do</h3>
```

Comment:
Good. This is a good example of modifier. It's the flavor text version of `index--digital`.